README.md

```diff
@@ -1 +1,12 @@
-# Investigation-Delete-Service
```

Comment:
I think we should use a template for the README.md of our projects. 
We can use the [project guidelines template](https://github.com/elsewhencode/project-guidelines/blob/master/README.sample.md).